include/mockturtle/io/aiger_reader.hpp

```diff
@@ -137,4 +141,101 @@ class aiger_reader : public lorina::aiger_reader
   mutable std::vector<signal<Ntk>> signals;
 };
 
+template<>
```

Comment:
I think that this is too strict. We could specialise on network types that do support `create_ro` and `create_ri`.